gpu/Dockerfile

```diff
@@ -19,17 +17,39 @@ RUN apt-get update && apt-get -y dist-upgrade \
   unzip \
   vim \
   wget \
-  ssh \
+  openssh-client \
   gnupg2 \
+  ca-certificates \
+  freetds-dev \
+  unixodbc-dev \
+  libssl-dev \
+  && apt-get clean \
   && rm -rf /var/lib/apt/lists/*
+ENV SHELL=/bin/bash \
+  LC_ALL=en_US.UTF-8 \
+  LANG=en_US.UTF-8 \
+  LANGUAGE=en_US.UTF-8 \
+  DEEPNOTE_PYTHON_KERNEL_ONLY=true
+
+RUN echo "en_US.UTF-8 UTF-8" > /etc/locale.gen && \
+locale-gen
 
+# Create virtual environment
+RUN python -m venv --system-site-packages ~/venv
```

Comment:
these `RUN` directives could be merged into a single layer

or even the following ones as well (to reduce 4 layers into 1)

Reply:
Since there is not much performance/image-size benefit, I would neglect it in favor of readability.